docs/blog/2023-20-11-thanoscon.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Introducing ThanosCon in Paris KubeCon EU 2024!
+date: "2023-11-20"
+author: The Thanos Team
+---
+
+<p align="center"><img height="40%" src="img/ThanosCon.png" width="40%"/></p>
+
+Hello Thanos Community! 👋
+
+Through recent years, the Thanos project has grown tremendously as an open-source scalable monitoring project based on Prometheus. Not only technically as a project, where we are continuously innovating on how to store and query metrics more efficiently, but also as a community with an ever-growing base of users, evident in our slack channels, docker pulls and (obviously!) GitHub stars.
+
+On behalf of the maintainers team behind Thanos, we are grateful to see this energy and passion towards building and improving Thanos, solving non-trivial challenges with truly open-source governance. We are regularly excited by the contributions and ideas brought by the community.
+
+To continue and celebrate this energy, we are extremely happy to announce the very first iteration of [ThanosCon](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/co-located-events/thanoscon/#about), as a KubeCon EU 2024 co-located event (Paris, 19th March)!
+
+This will be an event fully focussed on the Thanos project, where people will be able to learn from experts, network with maintainers and other users/community members and get into the world of open-source, scalable monitoring.
+
+The intention is to curate talks for an enjoyable, high-quality event, by and for Thanos users and contributors! We want the audience and community to connect and experience the excitement, love and friendliness of open-source, the values that led us from the beginning of the Thanos journey.
+
+Finally, we want you to learn from people running Thanos in production or building on top of it. We want you to walk away from this event, feeling excited about the future of Thanos and knowing how you can get involved!
+
+You might have heard about the Thanos project in various talks at popular conferences like KubeCon, FOSDEM, PromCon, PrometheusDays or ObservabilityDays. It's finally time for the one-off, dedicated, focused time during ThanosCon to celebrate, connect and learn!
+
+We are planning a lot of surprises for you. Will there be a SWAG? Will we celebrate a potential Thanos 1.0 release? Will you be able to talk to initial authors like Fabian Reinartz and Bartłomiej Płotka? Join us to learn, it will be fun for sure!
```

Comment:
```suggestion
We are planning a lot of surprises for you. Will there be a SWAG? Will we celebrate a potential Thanos 1.0 release? Will there be initial authors like Fabian Reinartz and Bartłomiej Płotka to learn from? Join us to learn; it will be fun for sure!
```